frontend/src/components/shared/Header.js

```diff
@@ -1,8 +1,43 @@
-
 import React from 'react'
+import { Box, Typography, Link } from '@mui/material'
 
 function Header() {
-    return <div>-header component placeholder-</div>
+    return (
+        <main>
+            <div>
+                <Box sx={{ display: "flex", flexDirection: "row", bgcolor: "primary.dark", p: 3 }}>
+                    <Box component="img"
+                        sx={{ height: 50, width: 45 }}
+                        src="/assets/bp-logo.png" />
+                    <Typography fontSize={20} color='primary.light' mt={0.5} ml={0.5}>blueprint</Typography>
```

Comment:
replace with this logo: https://drive.google.com/file/d/1fHsudAy8SiM1J1dfUp73e07BxWfhUQ12/view?usp=sharing

Reply:
add left margin

Reply:
make it link to home page

Reply:
the logo in the link doesnt have blue colors on the logo. 
![image](https://user-images.githubusercontent.com/61950298/173915788-d1acdc42-af4d-4526-b2f1-de196c52667f.png)